include/cppkafka/utils/buffered_producer.h

```diff
@@ -530,6 +530,7 @@ class CPPKAFKA_API BufferedProducer {
     // Members
     Producer producer_;
     QueueType messages_;
+    QueueType hi_pri_messages_;
```

Comment:
I'm not a fan of abbreviations. I'd rather this be "high_priority_messages_" rather than "hi_pri" which is not very self describing.